src/main/java/net/geant/nmaas/portal/service/impl/BulkApplicationServiceImpl.java

```diff
@@ -342,19 +342,13 @@ public List<BulkAppDetails> getAppsBulkDetails(BulkDeploymentView bulkDeployment
             AppInstance instance = instanceService.find(instanceId).orElseThrow();
 
             Map<String, String> configurationParameters = new HashMap<>();
-            if (!Objects.isNull(instance.getConfiguration())) {
```

Comment:
So we're not adding those anymore?

Reply:
Additional params contains configuration params